internal/asyncapi/account_details.go

```diff
@@ -113,8 +113,9 @@ func (d *accountDetails) getTopicDescription() error {
 	if err != nil {
 		return catchOpenAPIError(err)
 	}
-	if atlasEntityWithExtInfo.Entity.Attributes["description"] != nil {
-		d.channelDetails.currentTopicDescription = fmt.Sprintf("%v", atlasEntityWithExtInfo.Entity.Attributes["description"])
+	attributes := atlasEntityWithExtInfo.Entity.GetAttributes()
+	if attributes["description"] != nil {
```

Comment:
Is this dictionary guaranteed to contain "description"? It would panic if it didn't, right?

Reply:
in golang it will not panic but return a default value of that type (which would be nil)

Reply:
Hmm TIL. Maybe this would be more readable?
```suggestion
	if _, ok := attributes["description"]; ok {
```